packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -2021,218 +2021,14 @@ export function writeEndSegment(
-//   }
-//   // Find the boundary around the fallback. This is always the previous node.
-//   const suspenseNode = suspenseIdNode.previousSibling;
-
-//   if (!errorDigest) {
-//     // Clear all the existing children. This is complicated because
-//     // there can be embedded Suspense boundaries in the fallback.
-//     // This is similar to clearSuspenseBoundary in ReactDOMHostConfig.
-//     // TODO: We could avoid this if we never emitted suspense boundaries in fallback trees.
-//     // They never hydrate anyway. However, currently we support incrementally loading the fallback.
-//     const parentInstance = suspenseNode.parentNode;
-//     let node = suspenseNode.nextSibling;
-//     let depth = 0;
-//     do {
-//       if (node && node.nodeType === COMMENT_NODE) {
-//         const data = node.data;
-//         if (data === SUSPENSE_END_DATA) {
-//           if (depth === 0) {
-//             break;
-//           } else {
-//             depth--;
-//           }
-//         } else if (
-//           data === SUSPENSE_START_DATA ||
-//           data === SUSPENSE_PENDING_START_DATA ||
-//           data === SUSPENSE_FALLBACK_START_DATA
-//         ) {
-//           depth++;
-//         }
-//       }
-
-//       const nextNode = node.nextSibling;
-//       parentInstance.removeChild(node);
-//       node = nextNode;
-//     } while (node);
-
-//     const endOfBoundary = node;
-
-//     // Insert all the children from the contentNode between the start and end of suspense boundary.
-//     while (contentNode.firstChild) {
-//       parentInstance.insertBefore(contentNode.firstChild, endOfBoundary);
-//     }
-
-//     suspenseNode.data = SUSPENSE_START_DATA;
-//   } else {
-//     suspenseNode.data = SUSPENSE_FALLBACK_START_DATA;
-//     suspenseIdNode.setAttribute('data-dgst', errorDigest)
-//   }
-
-//   if (suspenseNode._reactRetry) {
-//     suspenseNode._reactRetry();
-//   }
-// }
-
-// function completeSegment(containerID, placeholderID) {
-//   const segmentContainer = document.getElementById(containerID);
-//   const placeholderNode = document.getElementById(placeholderID);
-//   // We always expect both nodes to exist here because, while we might
-//   // have navigated away from the main tree, we still expect the detached
-//   // tree to exist.
-//   segmentContainer.parentNode.removeChild(segmentContainer);
-//   while (segmentContainer.firstChild) {
-//     placeholderNode.parentNode.insertBefore(
-//       segmentContainer.firstChild,
-//       placeholderNode,
-//     );
-//   }
-//   placeholderNode.parentNode.removeChild(placeholderNode);
-// }
-
+// These are injected by the build system. See ReactDOMFizzClientInstructionSet.js.
 const completeSegmentFunction =
-  'function $RS(a,b){a=document.getElementById(a);b=document.getElementById(b);for(a.parentNode.removeChild(a);a.firstChild;)b.parentNode.insertBefore(a.firstChild,b);b.parentNode.removeChild(b)}';
+  '__FIZZ_INSTRUCTION_SET_COMPLETE_SEGMENT_FUNCTION__';
```

Comment:
does rollup interpolate string values when looking to replace plugs? I had assumed it was interpreting a token not just a string when doing things like `__DEV__` replacement but I guess not :)

Reply:
Yeah, that was a surprise to me as well but this was my bad idea originally 😅 Just refactored in a new commit to not replace string literals (which is also hard to intercept per @acdlite's comment)